src/UI/Factory.php

```diff
@@ -336,4 +337,62 @@ public function panel();
 	 **/
 	public function modal();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     Popovers can be used if space is scarce to display secondary information such as a preview of
+	 *     some item or to offer limited user interaction.
+	 *   composition: >
+	 *     Popovers consist of a layer displayed above all other content.
+	 *     The content of the Popover depends on the functionality it performs.
+	 *     A Popover MAY display a title above its content.
+	 *     All Popovers contain a pointer pointing from the Popover to the Triggerer of the Popover.
```

Comment:
To me the distinction between "the trigger" and "the entity that pulls the trigger" is valuable. I'm not emottionally attached to "triggerer", so if @atoedt proposes another word to replace "triggerer", I'm also fine.